client/state/sites/selectors/get-site-wordpress-seo-wizard-url.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Internal dependencies
+ */
+import getSiteAdminUrl from 'calypso/state/sites/selectors/get-site-admin-url';
+
+/**
+ * Returns a site's wp-admin WooCommerce Wizard plugin URL,
```

Comment:
We should replace WooCommerce with Yoast or WordPress SEO in this file

Reply:
yup! updating...